test/sharness/lib/test-lib-hashes.sh

```diff
@@ -0,0 +1,7 @@
+# this file defines several useful hashes used across the test codebase.
+# thus they can be defined + changed in one place
+
+HASH_WELCOME_DOCS="QmYwAPJzv5CZsnA625s3Xf2nemtYgPpHdWEz79ojWnPbdG"
+HASH_GATEWAY_ASSETS="QmXB7PLRWH6bCiwrGh2MrBBjNkLv3mY3JdYXCikYZSwLED"
+HASH_HELP_PAGE="QmY5heUM5qgRubMDD1og9fhCPA6QdkMp3QCwd4s7gJsyE7"
+HASH_EMPTY_DIR="QmUNLLsPACCz1vLxQVkXqqLX5R1X345qqfHbsf67hvA3Nn"
```

Comment:
It doesn't change so often, so I think manually is fine for now.
Last time they changed was in January.